rspack.embedding-sdk-package.config.js

```diff
@@ -75,7 +92,9 @@ const config = {
 
   plugins: [
     new rspack.EnvironmentPlugin({
+      IS_EMBEDDING_SDK: "true",
```

Comment:
Added it just in case if some shared code that is used by SDK relies on this (currently it's not)